src/setup-beam.js

```diff
@@ -17,19 +18,15 @@ async function main() {
   const otpVersion = await installOTP(otpSpec, osVersion)
 
   const elixirSpec = core.getInput('elixir-version', { required: false })
-  const shouldMixHex = core.getInput('install-hex', {
```

Comment:
This is actually just moving code around, to revert a previous decision that created more confusion that expected.